plugins/scheme-tpm-enacttrust/test/kvstores.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Note that these won't work for now because I haven't yet done https://github.com/veraison/veraison/issues/122.

At present, the provisioning pipeline serialises an EnactTrust trust anchor as:
```json
{
  "type": "TA_RAWPUBLICKEY",
  "value": {
    "enacttrust.ak-pub": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE6Vwqe7hy3O8Ypa+BUETLUjBNU3rEXVUyt9XHR7HJWLG7XTKQd9i1kVRXeBPDLFnfYru1/euxRnJM7H9UoFDLdA=="
  }
}
```

and an EnactTrust refval as:

```json
{
  "id": {
    "type": "TPM_ENACTTRUST",
    "parts": {
      "enacttrust-tpm.node-id": "ffffffff-ffff-ffff-ffff-ffffffffffff"
    }
  },
  "attributes": {
    "enacttrust-tpm.alg-id": 1,
    "enacttrust-tpm.digest": "h0KPxSKAPTEGXnvOPPA/5HUJZjHl4Hu9eg/eYMTPJcc="
  }
}
```

Reply:
Yeah, I actually ended up not using it directly. Will remove from commit.